### PR TITLE
Add support for OmniAuth 2.0

### PR DESCRIPTION
**Pull request by jkowens**
Hoping this helps the Devise team as they work on releasing an update to support OmniAuth 2.0. 

See: https://github.com/heartcombo/devise/issues/5326#issuecomment-758819662

### CODE REVIEW

**Comment by jkowens** [comment]
@BobbyMcWho would you be able to review this PR? Sorry I know you’ve been busy 😬

**Comment by BobbyMcWho** [comment]
@jkowens sure, sorry I don't get notifications in this repo on activity. I'll take a look today as soon as I can, and I'll figure out who I need to herd to get this released on rubygems.

**Comment by BobbyMcWho** [comment]
It looks like @mbleigh is the owner on rubygems

File: omniauth-openid.gemspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 require File.expand_path('../lib/omniauth-openid/version', __FILE__)
 
 Gem::Specification.new do |gem|
-  gem.add_dependency 'omniauth', '~> 1.0'
+  gem.add_dependency 'omniauth', ['>= 1.0', '< 3.0']
```
Comment (BobbyMcWho) [inline omniauth-openid.gemspec:5]: Let's just go with a pessimistic 2.0 here just in case
```suggestion
  gem.add_dependency 'omniauth', '~> 2.0'
```

Comment (BobbyMcWho) [reply omniauth-openid.gemspec:5]: Actually on second thought let's leave it permissive since there are no real breaking changes here.

**Comment by BobbyMcWho** [comment]
@jkowens Just need the version file update committed since it looks like the Gemfile.lock has this gem bumped to 2.0.0?

**Comment by jkowens** [comment]
@BobbyMcWho I think the reason for the version change in Gemfile.lock is because the gem was released as 2.0, but I don't think the Gemfile.lock had been updated.

**Comment by BobbyMcWho** [comment]
Ah you're correct, the main branch has v2.0.0 but it's never been released on rubygems.

**Comment by carlosantoniodasilva** [comment]
Thank you both for your work on this! :heart:

**Comment by BobbyMcWho** [comment]
I'll release this as 2.0.0 later, I need to do some history digging and see what has been merged since the last rubygems release was cut.

**Comment by BobbyMcWho** [comment]
@carlosantoniodasilva @jkowens I released it as v2.0.1 just to be safe.

**Comment by carlosantoniodasilva** [comment]
Perfect, thanks. I'll look into the devise updates next, hopefully this week still.

**Pull request merged**